src/Books/bookModels.js

```diff
@@ -1,15 +1,26 @@
-// const sequalize = require("sequalize")
+const sequalize = require("sequalize")
 
-// const userSchema = new sequalize.Schema ({
-//   bookName:{
-//     type: String,
-//     required: true,
-//     unique: true
-//   },
-//   bookAuthor:{
-//     type: String,
-//     required: true,
-//     unique: true
-//   },
-
-// });
+const userSchema = new sequalize.Schema ({
+  bookTitle:{
+    type: String,
+    required: true,
+    unique: true
+  },
+  bookAuthor:{
+    type: String,
+    required: true,
+    unique: true
+  },
+  bookImage:{
+    type: Image,
+    required:false,
+    unique: false
+  },
+  bookISBN:{
+    type: Number,
+    required:true,
```

Comment:
Maybe set this to false to begin with - may cause some issues due to the way that ISBN number is stored in the API

Reply:
great job Nick,

to benefit of sequelize use as e.g. 
title: {
        type: DataTypes.STRING,
        allowNull: false,
        unique: true,
    },

it will work definitely bus as suggestions  tables columns can  be same like google naming so we can change 


bookName into "title"
bookAuthor into "authors"

bookImage I think can be" String" as it is just URL no need to store the image FILE I daught that we can store file
the google books  JSON has one key called "imageLinks" which subdivided into "smallThumbnail"  "thumbnail" "small" "medium" "large" "extraLarge"
the look of how our website look will improve if we import medium to large.
so ask chris about it 

we can add googleID as Chris suggest and it is  just "id" on JSON
as chris suggested also REQUIRED and UNIQUE to be false is better 

as e.g. so we can fill books table with just title id and authors  with empty image, ISBN